src/Core/Auth/UserFeatures/Registration/Implementations/SendVerificationEmailForRegistrationCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿#nullable enable
+using Bit.Core.Auth.Models.Business.Tokenables;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Settings;
+using Bit.Core.Tokens;
+
+namespace Bit.Core.Auth.UserFeatures.Registration.Implementations;
+
+public class SendVerificationEmailForRegistrationCommand : ISendVerificationEmailForRegistrationCommand
```

Comment:
nit: would love to see a description on the command

Reply:
Done!